helpdesk_mgmt/migrations/13.0.1.2.3/post-migrate.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def remove_obsolete_external_id_and_record(cr, external_id):
+    _logger.info("Cleaning obsolete external ID: %s" % external_id)
+    module, name = external_id.split(".")
+    cr.execute(
+        """
+        SELECT id, res_id, model
+        FROM ir_model_data
+        WHERE "module" = %s
+            AND name = %s;
+    """,
+        (module, name),
+    )
+    imd_id, res_id, model = cr.fetchone()
+    cr.execute(
+        """
+        SELECT partner_id, product_id
+        FROM product_requestor_rel
+    """
+    )
+    table_name = model.replace(".", "_")
+    cr.execute(
+        """
+        DELETE FROM %s WHERE id = %s
+    """,
+        (table_name, res_id),
+    )
+    cr.execute(
+        """
+        DELETE FROM ir_model_data WHERE id = %s;
+    """,
+        imd_id,
+    )
+
+
+def migrate(cr, version):
+    remove_obsolete_external_id_and_record(
+        cr, "helpdesk_mgmt.helpesk_ticket_personal_rule"
+    )
```

Comment:
why not use openupgradelib.delete_records_safely_by_xml_id() method?

Reply:
Some people prefer not to depen on openupgrade on this "update-scripts"

Reply:
You could copy the code of the method hehe

Reply:
haha Copying? me? never :stuck_out_tongue: 

Just kidding, I will do that next time.